Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1120,7 +1112,8 @@ void ExecuteCommand(u32 command_0, u32 command_1, u32 command_2, u32 output_addr
   // to simulate the speed of a real disc drive
   if (!command_handled_by_thread)
   {
-    CoreTiming::ScheduleEvent(ticks_until_completion, s_finish_executing_command,
+    // This is an arbitrary delay - needs testing to determine the appropriate value
+    CoreTiming::ScheduleEvent(SystemTimers::GetTicksPerSecond() / 15000, s_finish_executing_command,
```

Comment:
Weird. GH didn't save my comment here, but we should extract this `/ 15000` into a constant (66.6 microseconds).